server/events/working_dir.go

```diff
@@ -256,6 +258,17 @@ func (w *FileWorkspace) GetPullDir(r models.Repo, p models.PullRequest) (string,
 	return dir, nil
 }
 
+// ListAllFiles return all files present in the repo.
+func (w *FileWorkspace) ListAllFiles(baseRepo models.Repo, p models.PullRequest, workspace string) ([]string, error) {
```

Comment:
Based on my suggestion above I don't think you'll need this function.

Reply:
Hi @lkysow, thanks for the review!

Just to be sure, I will need to scan the filesystem in `DetermineProjects()` to find all projects right?

Reply:
Yes but you don't necessarily need to use `git ls-tree` now do you? I was thinking you could use the golang filesystem packages.

Reply:
Sorry, busy week. Yes absolutely! I just wanted to make sure I understand correctly before changing the code.